resources/keb/values.yaml

```
@@ -330,6 +330,8 @@ oidc:
     orchestrations: orchestrationsAdmin
     viewer: runtimeViewer
 
+oidcs: []
```

instead, I would just have a new issuers list with empty default under the existing oidc config:

```suggestion
oidc:
  issuers: []
```